src/site/content/en/blog/periodic-background-sync/index.md

```diff
@@ -336,6 +335,13 @@ self.addEventListener('periodicsync', (event) => {
 });
 ```
 
+### Adding periodic background sync to an existing web app
+
+[This set of changes](https://github.com/GoogleChromeLabs/so-pwa/pull/11) were needed to add in
+periodic background sync to an [existing web app](https://so-pwa.firebaseapp.com/) that already had
+offline support. This example includes a number of helpful logging statements that describe the
```

Comment:
```suggestion
periodic background sync to an [existing PWA](https://so-pwa.firebaseapp.com/). This example includes a number of helpful logging statements that describe the
```

Reply:
Unless I'm missing something. 

What am I looking for at the end of that link? If you can't link to a specific question, can you tell the reader what they're looking for?

Reply:
Your change seems 👍